infra/modules/database/dms-instance.tf

```diff
@@ -12,43 +28,54 @@ resource "aws_dms_replication_instance" "simpler_db" {
   preferred_maintenance_window = "sun:10:30-sun:14:30"
   publicly_accessible          = false
   replication_instance_class   = "dms.t3.small"
-  replication_instance_id      = "db-replication-instance"
+  replication_instance_id      = "${var.environment_name}-db-replication-instance"
+  replication_subnet_group_id  = aws_dms_replication_subnet_group.subnet.id
 
   vpc_security_group_ids = [
-    data.aws_security_group.source_db.id
+    data.aws_security_group.dms.id
   ]
-
 }
 
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dms_endpoint
 resource "aws_dms_endpoint" "target_endpoint" {
+  endpoint_id                     = "${var.environment_name}-simpler-grants-target"
   certificate_arn                 = "arn:aws:dms:us-east-1:315341936575:cert:GWOIQRTIVQVRBL5ERMCKTUPHMM33MMDGIP57J4I"
   database_name                   = "app"
-  endpoint_id                     = "api-dev-primary"
   endpoint_type                   = "target"
   engine_name                     = "aurora-postgresql"
   kms_key_arn                     = aws_kms_key.dms_endpoints.arn
   secrets_manager_access_role_arn = aws_iam_role.dms_access.arn
   ssl_mode                        = "verify-ca"
-  secrets_manager_arn             = data.aws_secretsmanager_secret.db_password.arn
+  secrets_manager_arn             = data.aws_secretsmanager_secret.target_db.id
 }
 
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dms_endpoint
 resource "aws_dms_endpoint" "source_endpoint" {
   # checkov:skip=CKV2_AWS_49: This endpoint doesn't need SSL
+  endpoint_id                     = "${var.environment_name}-grants-gov-source"
   database_name                   = "tstgrnts"
-  endpoint_id                     = "hhs-source"
   endpoint_type                   = "source"
   engine_name                     = "oracle"
   kms_key_arn                     = aws_kms_key.dms_endpoints.arn
   ssl_mode                        = "none"
   secrets_manager_access_role_arn = aws_iam_role.dms_access.arn
-  secrets_manager_arn             = data.aws_secretsmanager_secret.source_db.arn
+  secrets_manager_arn             = data.aws_secretsmanager_secret.source_db.id
```

Comment:
It says "arn" but recommendations I saw online said to use the `id` instead 🤷🏼‍♂️ 🤷🏼‍♀️ 🤷🏼